src/main/perl/Blockdevices.pm

```diff
@@ -465,4 +466,31 @@ sub has_filesystem
     return $f =~ m{\s$fsregex\s+file}i;
 }
 
+=pod
+
+=head2 get_uuid
+
+Fetches the (PART)UUID of the device with blkid. Returns undef when not found.
+type can be an empty string or 'PART'
```

Comment:
Why use a string for a boolean? Is there some advantage to the $type taking other values in future?